rpmlint/checks/FileDigestCheck.py

```diff
@@ -198,11 +210,30 @@ def _matches_pkg(self, digest_group, pkg):
 
         return False
 
+    def _is_path_covered_by_digests(self, path):
+        """
+        Return true if there is a digest configuration that covers
+        provided file at given path.
+        """
+        parts = list(path.parts) + [None]
+        node = self.digest_configuration_locations
+        for part in parts:
+            if None in node:
+                return True
+            if part not in node:
```

Comment:
This could be a tad more efficient by doing:

```
try:
    node = node[part]
except KeyError:
    return False
```

That avoids two identical dictionary lookups

Reply:
The code is different now, but I wanted to not include exception machinery that tends to be slower than a normal dictionary lookup.